src/frontend/screens/ObservationCreate/index.tsx

```diff
@@ -36,8 +38,48 @@ const m = defineMessages({
     defaultMessage: 'What is happening here?',
     description: 'Placeholder for description/notes field',
   },
+  noGpsTitle: {
+    id: 'screens.ObservationCreate.noGpsTitle',
+    defaultMessage: 'No GPS signal',
+    description: 'Title of dialog when trying to save with no GPS coords',
+  },
+  noGpsDesc: {
+    id: 'screens.ObservationCreate.noGpsDesc',
+    defaultMessage:
+      'This observation does not have a location. You can continue waiting for a GPS signal, save the observation without a location, or enter coordinates manually',
+    description: 'Description in dialog when trying to save with no GPS coords',
+  },
+  weakGpsTitle: {
+    id: 'screens.ObservationCreate.weakGpsTitle',
+    defaultMessage: 'Weak GPS signal',
+    description: 'Title of dialog when trying to save with low GPS accuracy.',
+  },
+  weakGpsDesc: {
+    id: 'screens.ObservationCreate.weakGpsDesc',
+    defaultMessage:
+      'GPS accuracy is low. You can continue waiting for better accuracy, save the observation with low accuracy, or enter coordinates manually',
+    description:
+      'Description in dialog when trying to save with low GPS accuracy.',
+  },
+  saveAnyway: {
+    id: 'screens.ObservationCreate.saveAnyway',
+    defaultMessage: 'Save',
+    description: 'Button to save regardless of GPS state',
+  },
+  manualEntry: {
+    id: 'screens.ObservationCreate.manualEntry',
+    defaultMessage: 'Manual Coords',
+    description: 'Button to manually enter GPS coordinates',
+  },
+  keepWaiting: {
+    id: 'screens.ObservationCreate.keepWaiting',
+    defaultMessage: 'Continue waiting',
+    description: 'Button to cancel save and continue waiting for GPS',
+  },
 });
 
+const MINIMUM_ACCURACY = 10;
```

Comment:
think this should be renamed to `MAXIMUM_ACCURACY` or something similar? See other comment where this is used, but this should be representing the maximum value we allow without asking the user about potentially inserting a manual one.

Reply:
https://github.com/digidem/comapeo-mobile/pull/702/commits/9ed6b36b9c35fd46ccb358820f0d0980cfc3aab7